tools/cli/commands/connect.py

```diff
@@ -277,6 +277,13 @@ def connect_and_check():
             return
         if remaining_reconnects == 0:
             return
+        # Before we try to reconnect, check to see if the VM is still running.
+        status, metadata_items = utils.describe_instance(
+            args, gcloud_compute, instance)
+        if status != 'RUNNING':
```

Comment:
Now that we are using 'RUNNING' in two different places, perhaps we should define a constant for it.